Ginger/GingerCoreCommon/Repository/BusinessFlowLib/BusinessFlow.cs

```diff
@@ -87,6 +87,41 @@ public enum eSource
             Gherkin     // From Gherking Feature file
         }
 
+        public static partial class Fields
+        {
+            public static string Active = "Active";
+            public static string Mandatory = "Mandatory";
+            public static string Name = "Name";
+            public static string Description = "Description";
+            public static string RunDescription = "RunDescription";
+            public static string Status = "Status";
+            public static string Activities = "Activities";
+            public static string Variables = "Variables";
+            public static string RunStatus = "RunStatus";
+            public static string Elapsed = "ElapsedSecs";
+            public static string Platforms = "Platforms";
+            public static string EnableActivitiesVariablesDependenciesControl = "EnableActivitiesVariablesDependenciesControl";
+            public static string ActivitiesGroups = "ActivitiesGroups";
+            public static string AutomationPrecentage = "AutomationPrecentage";
+            public static string ExternalID = "ExternalID";
+            public static string PublishStatus = "PublishStatus";
+            public static string Source = "Source";
+        }
+
+        bool mSelectedForConversion;
+        public bool SelectedForConversion
```

Comment:
rename 'SelectedForConversion' to 'Selected' so we will be able to use it for other pages as well